assets/sass/patterns/atoms/definition-list.scss

```diff
@@ -1,5 +1,57 @@
 @import "../../definitions";
 
+.definition-list {
+  @include margin(0);
+  @include margin(24, "top");
```

Comment:
This can done as margin(24,0,0);

Reply:
The margin is now added in a single row.